crm-project/src/main/java/CRM/controller/facades/UserFacade.java

```diff
@@ -90,4 +96,30 @@ public Response getAllInBoard(Long boardId) {
                     .build();
         }
     }
+
+    public Response addUserToBoard(Long userId, Long boardId) {
+        try {
+            Validations.validate(userId, Regex.ID.getRegex());
+            Validations.validate(boardId, Regex.ID.getRegex());
+
+            return new Response.Builder()
+                    .data(userService.addUserToBoard(userId, boardId))
```

Comment:
replace -> userService.addUserToBoard(userId, boardId)
to UserInBoardDTO.getUserInBoardFromDB(userService.addUserToBoard(userId, boardId))

Reply:
@shailevi23 solved.